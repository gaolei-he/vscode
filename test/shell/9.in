48
/ca
/dda/da/abd
/cad/cda/bab
/bab
/aac/ad
/d/aba/bbc
/dad/c
/bba/d/bd
/a/bc/ccb
/ab/a
/cd/cc/cc
/bab/b/ad
/a/db
/a
/d/ac
/ac
/c/ca
/cda/a
/dd
/b
/c
/da/dbc/acd
/acc
/dcb/dcd/bdd
/bba
/abc
/b
/db
/ad/da/ccb
/cd/bbb/b
/dad/cdc
/ca
/ad/d
/bba/d/acb
/ca/cd
/b/cd/aac
/b
/b/d/c
/bb/ddb/d
/a
/db/b
/db
/b/cc
/c/aab
/bbc
/ad/c/baa
/bc/dd
/dcb/abb
54
cd abc
pwd
pwd
ls
pwd
pwd
cd ..
cd ..
cd ..
ls
pwd
ls
pwd
pwd
cd ..
cd ..
pwd
ls
ls
cd 
pwd
ls
pwd
ls
ls
cd .
cd ..
ls
cd ..
pwd
ls
cd .
cd .
cd ..
pwd
cd ..
ls
cd ..
pwd
cd .
ls
pwd
pwd
ls
pwd
cd .
pwd
pwd
cd .
cd ..
ls
ls
cd ..
pwd
