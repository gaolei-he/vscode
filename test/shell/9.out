/abc
/abc
/abc
/abc




/
a
aac
ab
abc
ac
acc
ad
b
bab
bb
bba
bbc
bc
c
ca
cad
cd
cda
d
da
dad
db
dcb
dd
dda
/
a
aac
ab
abc
ac
acc
ad
b
bab
bb
bba
bbc
bc
c
ca
cad
cd
cda
d
da
dad
db
dcb
dd
dda
a
aac
ab
abc
ac
acc
ad
b
bab
bb
bba
bbc
bc
c
ca
cad
cd
cda
d
da
dad
db
dcb
dd
dda
a
aac
ab
abc
ac
acc
ad
b
bab
bb
bba
bbc
bc
c
ca
cad
cd
cda
d
da
dad
db
dcb
dd
dda
/
a
aac
ab
abc
ac
acc
ad
b
bab
bb
bba
bbc
bc
c
ca
cad
cd
cda
d
da
dad
db
dcb
dd
dda
/
a
aac
ab
abc
ac
acc
ad
b
bab
bb
bba
bbc
bc
c
ca
cad
cd
cda
d
da
dad
db
dcb
dd
dda
/
a
aac
ab
abc
ac
acc
ad
b
bab
bb
bba
bbc
bc
c
ca
cad
cd
cda
d
da
dad
db
dcb
dd
dda
/
/
a
aac
ab
abc
ac
acc
ad
b
bab
bb
bba
bbc
bc
c
ca
cad
cd
cda
d
da
dad
db
dcb
dd
dda
/
/
/
a
aac
ab
abc
ac
acc
ad
b
bab
bb
bba
bbc
bc
c
ca
cad
cd
cda
d
da
dad
db
dcb
dd
dda
a
aac
ab
abc
ac
acc
ad
b
bab
bb
bba
bbc
bc
c
ca
cad
cd
cda
d
da
dad
db
dcb
dd
dda
/
